docs/CONFIG-VARS.md

```diff
@@ -228,6 +228,7 @@ When setting V4_CFG_TLS_MODE to a value other than "disabled" and no V4_CFG_TLS_
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
+| V4_CFG_CLUSTER_NODE_POOL_MODE | What mode of cluster node pool to use | string | "standard" | false | [standard, minimal] | viya |
```

Comment:
something is wrong with the editing change here. Look at it in rich diff. Many elastic search lines are listed as crossed out.

Reply:
I took a look at the page and don't see any incorrect formatting. https://github.com/sassoftware/viya4-deployment/blob/minimal_deploy/docs/CONFIG-VARS.md

Reply:
Yes that link looks good to me too. It's when I choose rich diff that things are not rendering correctly. I use rich diff all the time and it always renders correctly. When I've seen issue like the this in the past it was a syntax error. 

![image](https://user-images.githubusercontent.com/44618553/123309104-7f6dab80-d4f2-11eb-96ea-32463742cc49.png)